app/controllers/students_controller.rb

```diff
@@ -8,6 +8,15 @@ def update
     end
   end
 
+  def index
+    @students = current_admin.current_cohort.students
+  end
+
+  def internships
+    @student = Student.find(params[:student_id])
+    @internships = @student.cohort.internships_sorted_by_interest(@student)
```

Comment:
you've got some redundant info here: `@student.cohort.internships_sorted_by_interest(@student)`. when you call a method on `@student`, you should need to pass in that same object as an argument. instead, make a `Student#internship_rankings` method (or something like that).